plugins/lsp/conform-nvim.nix

```diff
@@ -0,0 +1,89 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.conform-nvim;
+  helpers = import ../helpers.nix {inherit lib;};
+in {
+  options.plugins.conform-nvim =
+    helpers.extraOptionsOptions
+    // {
+      enable = mkEnableOption "conform-nvim";
+
+      package = helpers.mkPackageOption "conform-nvim" pkgs.vimPlugins.conform-nvim;
+
+      # Map of filetype to formatters
+      formattersByFt =
+        helpers.defaultNullOpts.mkNullable types.attrs
+        "see documentation"
+        ''
+          Map of filetype to formatters
+          lua = { "stylua" },
+          Conform will run multiple formatters sequentially
+          python = { "isort", "black" },
+          Use a sub-list to run only the first available formatter
+          javascript = { { "prettierd", "prettier" } },
+          Use the "*" filetype to run formatters on all filetypes.
+          ["*"] = { "codespell" },
+          Use the "_" filetype to run formatters on filetypes that don't
+          have other formatters configured.
+          ["_"] = { "trim_whitespace" },
+        '';
+
+      # If this is set, Conform will run the formatter on save.
+      # It will pass the table to conform.format().
+      # This can also be a function that returns the table.
+      # See :help conform.format for details.
+      formatOnSave = {
+        lspFallback =
+          helpers.defaultNullOpts.mkBool true
+          "See :help conform.format for details.";
+        timeoutMs =
+          helpers.defaultNullOpts.mkNullable (types.int) 1000
+          "See :help conform.format for details.";
+      };
+
+      # If this is set, Conform will run the formatter asynchronously after save.
+      # It will pass the table to conform.format().
+      # This can also be a function that returns the table.
+      formatAfterSave = {
+        lspFallback =
+          helpers.defaultNullOpts.mkBool true
+          "See :help conform.format for details.";
+      };
+
+      # Set the log level. Use `:ConformInfo` to see the location of the log file.
+      # logLevel = "vim.log.levels.${
+      #   helpers.defaultNullOpts.mkEnumFirstDefault ["ERROR" "DEBUG" "INFO" "TRACE" "WARN" "OFF"] "Set the log level. Use `:ConformInfo` to see the location of the log file."
+      # }";
+
+      # Conform will notify you when a formatter errors
+      notifyOnError =
+        helpers.defaultNullOpts.mkBool true
+        "Conform will notify you when a formatter errors";
+
+      # Custom formatters and changes to built-in formatters
```

Comment:
not needed

Reply:
Are you asking me to remove the `notifyOnError` option?

Reply:
No the nix comment, it is redudant with the string under it